src/actions/getSaleStatus.ts

```diff
@@ -1,26 +1,32 @@
 import * as ethers from "ethers";
-import { WolfSale } from "../contracts/types";
+import { AirWild2Sale } from "../contracts/types";
 
 import { SaleStatus } from "../types";
 
-export const getSaleStatus = async (contract: WolfSale) => {
+export const getSaleStatus = async (contract: AirWild2Sale) => {
   const saleStarted = await contract.saleStarted();
 
   if (!saleStarted) return SaleStatus.NotStarted;
 
   const currentBlock = await contract.provider.getBlockNumber();
   const startBlock = await contract.saleStartBlock();
-  const duration = await contract.privateSaleDuration();
+  const duration = await contract.mintlistDurations(
+    await contract.currentMerkleRootIndex()
+  );
 
   if (ethers.BigNumber.from(currentBlock).gt(startBlock.add(duration))) {
```

Comment:
The mint list doesn't switch over automatically, so what's the point of even checking the duration?